templates/flavors/vpcless/kustomization.yaml

```
@@ -0,0 +1,37 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../default
+
+patches:
+  - path: deleteLinodeVPC.yaml
```

could we inline this?
```suggestion
  - target:
      kind: LinodeVPC
    patch: |
      $patch: delete
      apiVersion: infrastructure.cluster.x-k8s.io/v1alpha1
      kind: LinodeVPC
      metadata:
        name: ${VPC_NAME:=${CLUSTER_NAME}}
```

---

Updated.